crypto-ffi/src/ciphersuite.rs

```diff
@@ -88,82 +86,22 @@ pub fn ciphersuite_default() -> Ciphersuite {
     Ciphersuite::default()
 }
 
-#[derive(Debug, Clone, derive_more::From, derive_more::Into)]
-#[cfg_attr(target_family = "wasm", wasm_bindgen)]
-#[cfg_attr(not(target_family = "wasm"), derive(uniffi::Object), uniffi::export(Debug))]
-pub struct Ciphersuites(HashSet<Ciphersuite>);
-
-#[cfg(target_family = "wasm")]
-pub(crate) type CiphersuitesMaybeArc = Ciphersuites;
-
-#[cfg(not(target_family = "wasm"))]
-pub(crate) type CiphersuitesMaybeArc = std::sync::Arc<Ciphersuites>;
-
-impl Ciphersuites {
-    pub fn iter(&self) -> impl Iterator<Item = Ciphersuite> {
-        self.0.iter().copied()
-    }
-}
+pub(crate) type Ciphersuites = Vec<Ciphersuite>;
 
-#[cfg_attr(target_family = "wasm", wasm_bindgen)]
+/// Helper function to convert a list of integers into a list of ciphersuites
+#[cfg_attr(target_family = "wasm", wasm_bindgen(js_name = ciphersuitesFromU16s))]
```

Comment:
Do we need this function exposed? What is the intended usage? I would rather that clients are not aware of the internal representation of ciphersuite values and that they just use proper enums.